### DISCUSSION TITLE
Does Carousel support youtube?

### DISCUSSION THREAD

**Discussion by colormango** [fancyapps/ui#238] [Q&A]
If I have both Images and YouTube videos, how can I use them on Carousel?
For example:

div class="carousel__slide"
img data-lazy-src="https://lipsum.app/id/46/1600x1200" /
/div
div class="carousel__slide"
img data-lazy-src="https://lipsum.app/id/47/1600x1200" />
/div
div class="carousel__slide"
img data-lazy-src="https://www.youtube.com/embed/-Y9nTVmNdyU" /
/div

**Comment by fancyapps** [comment]
Hi,

Sorry, it is not currently supported, but it is planned for future releases. For now, you could implement it using `selectSlide` event.
